Makefile

```diff
@@ -80,7 +80,7 @@ build/cache:
 
 .k0sbuild.docker-image.k0s: build/Dockerfile embedded-bins/Makefile.variables | build/cache
 	docker build --rm \
-		--build-arg BUILDIMAGE=golang:$(go_version)-alpine \
```

Comment:
Unit tests on ARM fail with segfault on alpine. Temporarily switch it to Ubuntu as we can fix it in a separate PR.